docs/preview/yamlgettingstarted-triggers.md

```diff
@@ -32,8 +34,41 @@ trigger:
 
 Note, path filters are only supported for Git repositories in VSTS.
 
-## CI is opt-out
+### CI is opt-out
 
 Continuous integration builds can be turned off by specifying `trigger: none`
 
 Optionally, the triggers can be managed from the web definition editor, on the Triggers tab.
+
+## Pull request triggers
+
+Pull request builds are on by default for all branches. Optionally an explicit list of branch filters can
+be specified in the YAML file.
+
+When a PR is updated, the YAML file from the target branch is used to evaluate the branch filters.
+
+And the workflow defined in the YAML file from the merge branch, is used to execute the build.
+
+### Simple PR trigger syntax
+
+A simple list of branches can be specified in the file, to control which branches trigger a PR build.
+
+For example, a simple list of inclusive branch filters may look like:
+
+```yaml
+trigger:
+- master
+- releases/*
+```
+
+### Full PR trigger syntax
+
+For more control, an alternative PR syntax is available:
+
+```yaml
+pr:
+  forks: true # whether to build PRs from forks
```

Comment:
we might actually want the forks bool to live in the UI

Reply:
What's the benefit to not permitting this to be configured via code? I can certainly understand not permitting secret access, but if I branch my code base for a release, I don't want to have to remember to go enable the forks option.

Reply:
this bool would not be branch specific